rdt/transformers/__init__.py

```diff
@@ -35,18 +46,44 @@ def _import_addons():
             for transformer in transformers:
                 module = transformer.rsplit('.', 1)[0]
                 if module not in sys.modules:
-                    importlib.import_module(module)
+                    module = importlib.import_module(module)
+                    for name, obj in inspect.getmembers(module):
+                        # Inject the addon transformer into this scope
+                        if inspect.isclass(obj) and issubclass(
+                                obj, BaseTransformer) and obj is not BaseTransformer:
+                            globals()[name] = obj
+                            __all__.append(name)
```

Comment:
I think that this is a cleaner way to achieve that, we already have the name of the addons class just we were skipping it. 
```python3
            for transformer in transformers:
                module, class_name = transformer.rsplit('.', 1)
                if module not in sys.modules:
                    module = importlib.import_module(module)
                    globals()[name] = getattr(module, class_name)
                    __all__.append(name)
```